be/src/util/lru_cache.cpp

```diff
@@ -294,9 +294,8 @@ void LRUCache::_evict_one_entry(LRUHandle* e) {
     _usage -= e->charge;
 }
 
-Cache::Handle* LRUCache::insert(const CacheKey& key, uint32_t hash, void* value, size_t charge,
-                                void (*deleter)(const CacheKey& key, void* value), CachePriority priority,
-                                size_t value_size) {
+Cache::Handle* LRUCache::insert(const CacheKey& key, uint32_t hash, void* value, size_t value_size, size_t charge,
+                                void (*deleter)(const CacheKey& key, void* value), CachePriority priority) {
```

Comment:
If the value is self-descriptive, how about removing the value_size in the lru_cache because it is really unnecessary for a underly cache? It can reduce memory if `value_size=charge` in many cases. If `value_size!=size`, the caller can get the value size from the pointer.

Reply:
Good suggestion. Removing it can save metadata memory. However, we won't change it in this PR. The purpose of this PR is to enable PageCache to use the StarCache interface.